### DISCUSSION TITLE
Add conditional style on Dynamic Columns in Datatables

### DISCUSSION THREAD

**Discussion by prabinsh** [primefaces/community#562] [PrimeVue] [answered]
How can I apply conditional styles or templates to dynamic columns in a datatable? I need to show/hide columns dynamically, and these columns include templates.

**Comment by tugcekucukoglu** [comment, marked as answer]
This demo might be starter: https://primevue.org/datatable/#conditional_style

**Comment by prabinsh** [comment]
```
<Column v-for="(col, index) of selectedColumns" :field="col.field" :header="col.header"
                :key="col.field + '_' + index" sortable :showFilterMatchModes="false">
                <template v-if="col.field === 'count'" #body="{ data }">
                    <Button label="Count" class="p-button-rounded p-button-text mr-2 mb-2"
                        @click="Count(data.count) : null" />
                </template>
</Column>
```

Thanks @tugcekucukoglu. I am posting this just in case somebody is looking for the solution as well. The code basically checks if the field is Count and applies templating. I extended this to the other columns with if.
